src/OmnichannelChatSDK.ts

```diff
@@ -484,46 +510,91 @@ class OmnichannelChatSDK {
+            return downloadedFile;
+        }
+        catch(e) {
+            this.scenarioMarker.failScenario(TelemetryEvent.DownloadFileAttachment);
+            return this.conversation!.downloadFile(fileMetadata);
+        }
     }
 
     public async emailLiveChatTranscript(body: IChatTranscriptBody): Promise<any> { // eslint-disable-line @typescript-eslint/no-explicit-any
         const emailTranscriptOptionalParams: IEmailTranscriptOptionalParams = {};
-        if (this.authenticatedUserToken) {
-            emailTranscriptOptionalParams.authenticatedUserToken = this.authenticatedUserToken;
+
+        this.scenarioMarker.startScenario(TelemetryEvent.EmailLiveChatTranscript, {
+            RequestId: this.requestId,
+            ChatId: this.chatToken.chatId as string
+        });
+
+        try {
+
+            if (this.authenticatedUserToken) {
+                emailTranscriptOptionalParams.authenticatedUserToken = this.authenticatedUserToken;
+            }
+            const emailRequestBody = {
+                ChatId: this.chatToken.chatId,
+                EmailAddress: body.emailAddress,
+                DefaultAttachmentMessage: body.attachmentMessage,
+                CustomerLocale: body.locale
+            };
+
+            this.scenarioMarker.completeScenario(TelemetryEvent.EmailLiveChatTranscript);
+
+            return this.OCClient.emailTranscript(
+                this.requestId,
+                this.chatToken.token,
+                emailRequestBody,
+                emailTranscriptOptionalParams);
+        }
+
+        catch(e) {
+            this.scenarioMarker.failScenario(TelemetryEvent.EmailLiveChatTranscript);
         }
-        const emailRequestBody = {
-            ChatId: this.chatToken.chatId,
-            EmailAddress: body.emailAddress,
-            DefaultAttachmentMessage: body.attachmentMessage,
-            CustomerLocale: body.locale
-        };
-        return this.OCClient.emailTranscript(
-            this.requestId,
-            this.chatToken.token,
-            emailRequestBody,
-            emailTranscriptOptionalParams);
     }
 
     public async getLiveChatTranscript(): Promise<any> { // eslint-disable-line @typescript-eslint/no-explicit-any
         const getChatTranscriptOptionalParams: IGetChatTranscriptsOptionalParams = {};
-        if (this.authenticatedUserToken) {
-            getChatTranscriptOptionalParams.authenticatedUserToken = this.authenticatedUserToken;
+
+        this.scenarioMarker.startScenario(TelemetryEvent.GetLiveChatTranscript, {
+            RequestId: this.requestId,
+            chatId: this.chatToken.chatId as string
+        });
+
+        try {
+            if (this.authenticatedUserToken) {
+                getChatTranscriptOptionalParams.authenticatedUserToken = this.authenticatedUserToken;
+            }
+
+            this.scenarioMarker.completeScenario(TelemetryEvent.EmailLiveChatTranscript);
```

Comment:
Same issue as with `emailLiveChatTranscript()`

Reply:
Scenario should be `TelemetryEvent.GetLiveChatTranscript` and not `TelemetryEvent.EmailLiveChatTranscript`